src/app/folder/folderlist.component.ts

```diff
@@ -43,6 +50,8 @@ class FolderNode {
 export class FolderListComponent {
     selectedFolder = 'Inbox';
     dropFolderId: number;
+    dropAboveOrBelowOrInside: DropPosition = 0;
```

Comment:
Shouldn't this be `... = DropPosition.NONE`?

Reply:
Indeed it should :+1:  Fixed now!